stack-graphs/src/graph.rs

```diff
@@ -535,17 +535,24 @@ impl NodeID {
 #[repr(C)]
 pub enum Node {
     DropScopes(DropScopesNode),
-    ExportedScope(ExportedScopeNode),
-    InternalScope(InternalScopeNode),
     JumpTo(JumpToNode),
     PopScopedSymbol(PopScopedSymbolNode),
     PopSymbol(PopSymbolNode),
     PushScopedSymbol(PushScopedSymbolNode),
     PushSymbol(PushSymbolNode),
     Root(RootNode),
+    Scope(ScopeNode),
 }
 
 impl Node {
+    #[inline(always)]
+    pub fn is_exported(&self) -> bool {
```

Comment:
nit: Can this be `is_exported_scope` to clarify that it only returns true for scope nodes, and not for clickable def/ref nodes?